addon/mixins/route.js

```diff
@@ -160,35 +172,24 @@ export default Ember.Mixin.create({
     delete options.perPage;
     delete options.modelPath;
 
-    this.set('_perPage', perPage);
-    this.set('_modelPath', modelPath);
-    this.set('_extraParams', options);
-
-    var requestPayloadBase = {};
-    requestPayloadBase[this.get('perPageParam')] = perPage;
-    requestPayloadBase[this.get('pageParam')] = startingPage;
```

Comment:
I need to make sure the initial value of `startingPage` is being used in the first request. It's not as of this commit. I will also add a test so this failure is apparent.